src/events/__tests__/types.test.tsx

```diff
@@ -1,9 +1,36 @@
 import "../../../jest.setup"
-import { Point } from "../types"
+import { Point2D } from "../../types/geometry"
+
+function relativeTo(idOrElem: string | HTMLElement) {
```

Comment:
This function was only used by tests, but as part of the old `Point` namespace couldn't be codesplit. So here I've refactored it out into the relative test.